aws/onboarding-scripts/byok8s/readme.md

```diff
@@ -0,0 +1,120 @@
+# Script to setup EKS cluster
```

Comment:
1. Please rename the file to `README.md`.
2. Change the folder structure to `aws/onboarding/byok8s/python/

Reply:
Reasoning: https://docs.google.com/document/d/10T7ZVEOUR3wN78bd7QlpCC5KJvsEJcvODNGAyy0cneg/edit?disco=AAABKWUm_LY

Thinking of this kind of a structure: `/{cloud}/{workflow}/{artifact}/{technology}/`